components/court/JurorsTable.tsx

```diff
@@ -32,44 +34,55 @@ const columns: TableColumn[] = [
     accessor: "status",
     type: "text",
   },
-  {
-    header: "",
-    accessor: "button",
-    type: "component",
-    width: "150px",
-  },
```

Comment:
can this be removed?

Reply:
Wanted to check with Jorn, we might keep it